### PR TITLE
Publish 6.0.0rc1 using the 'rc' label (take 2)

### PR DESCRIPTION
**Pull request by nicoddemus**
Follow up to https://github.com/conda-forge/pytest-feedstock/pull/119

### CODE REVIEW

**Comment by nicoddemus** [comment]
@conda-forge-admin, please rerender

**Comment by nicoddemus** [comment]
The rerender looks right to me, can you please double check @ocefpaf ?

File: .ci_support/linux_aarch64_python3.6.____73_pypy.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ cdt_name:
 channel_sources:
 - conda-forge
 channel_targets:
-- conda-forge main
+- conda-forge rc
```
Comment (ocefpaf) [inline .ci_support/linux_aarch64_python3.6.____73_pypy.yaml:10]: That is what we were missing!

Comment (nicoddemus) [reply .ci_support/linux_aarch64_python3.6.____73_pypy.yaml:10]: Awesome, thanks!

**Pull request merged**